OHM-Trade-Agent-v1/app/opip/data_platform/health.py

```
@@ -23,52 +30,311 @@ def _required_stream_readiness(
+            """
+            SELECT count(*), count(DISTINCT sync_fingerprint), max(sync_fingerprint)
+            FROM ops.required_stream
+            """
+        )
+        policy_count, fingerprint_count, current_fingerprint = cursor.fetchone()
+
+    if int(policy_count) == 0:
+        return MaintenanceInput(None, None, True)
+
+    # Validate complete policy synchronization
+    policy_valid, expected_count, actual_count = _validate_policy_sync(connection)
```

Ruff (0.16.3)

[warning] Unpacked variable `expected_count` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)